ksql-common/src/main/java/io/confluent/ksql/serde/KeyFormat.java

```diff
@@ -37,10 +37,11 @@ public static KeyFormat nonWindowed(final FormatInfo format) {
   public static KeyFormat windowed(
       final Format format,
       final Optional<String> avroSchemaName,
+      final Optional<Character> delimiter,
```

Comment:
Feels like this should be:

```java
  public static KeyFormat windowed(
         final FormatInfo format,
         final WindowInfo windowInfo
  ) {} 
```

So that we don't need to keep changing it when `FormatInfo` changes.